### PR TITLE
sha*sum: add --quiet option example

### PR DESCRIPTION
**Pull request by Agno94**
- [ ] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/sha256sum.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Read a file of SHA256 sums and verify all files have matching checksums:
 
 `sha256sum -c {{filename.sha256}}`
+
+- Verify all files have matching chesksums but don't print OK for succefully verified file:
```
Comment (navarroaxel) [inline pages/common/sha256sum.md:17]: Maybe this is better:
```suggestion
- Verify all files have matching chesksums but don't show informative messages:
```

File: pages/common/sha256sum.md [hunk 2]
```diff
@@ -13,3 +13,7 @@
 - Read a file of SHA256 sums and verify all files have matching checksums:
 
 `sha256sum -c {{filename.sha256}}`
```
Comment (navarroaxel) [inline pages/common/sha256sum.md:15]: Could we use the long argument format:
```suggestion
`sha256sum --check {{filename.sha256}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
I feel like we should include an example about how to create the `filename.sha256`. Maybe before the `--check` example:

`sha256sum {{filename1}} {{filename2}} > {{filename.sha256}}`

File: pages/common/shasum.md [hunk 3]
```diff
@@ -18,6 +18,10 @@
 
 `shasum --check {{list_file}}`
 
+- Check a list of sums but don't print OK for each successfully verified file:
+
+`shasum -c -q {{list_file}}`
```
Comment (navarroaxel) [inline pages/common/shasum.md:23]:
```suggestion
`shasum --check --quiet {{list_file}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/sha256sum.md (hunk 2 above)
Comment (Agno94) [reply pages/common/sha256sum.md:15]: The short argument was used in the previous line. Should I change that one too?

Comment (navarroaxel) [reply pages/common/sha256sum.md:15]: Yeah, please.

File: pages/common/sha256sum.md (hunk 1 above)
Comment (Agno94) [reply pages/common/sha256sum.md:17]: Do you think it's clear that `informative messages` means `OK` on successful file verification

**Comment by Agno94** [comment]
> I feel like we should include an example about how to create the `filename.sha256`. Maybe before the `--check` example:
> 
> `sha256sum {{filename1}} {{filename2}} > {{filename.sha256}}`

What about recursive glob pattern like:
`sha256sum ** > {{filename.sha256}}`

File: pages/common/sha256sum.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/sha256sum.md:21]: @sbrl ?

Comment (sbrl) [reply pages/common/sha256sum.md:21]: Hrm. What about this?
```suggestion
- Only show a message for files for which verification fails:
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/sha1sum.md [hunk 4]
```diff
@@ -10,6 +10,14 @@
 
 `sha1sum {{filename1}} {{filename2}}`
 
+- Calculate and save the list of SHA1 checksums to a file:
+
+`sha256sum {{filename1}} {{filename2}} > {{filename.sha1}}`
```
Comment (joshuatcasey) [inline pages/common/sha1sum.md:15]: Should this be a sha1sum command?

Note: similar examples for other sha*sum commands seem to have the wrong number.

**Review by sbrl — APPROVED** [review]
Thanks, @Agno94!

**Pull request merged**